plugins/orchestrator/src/components/ExecuteWorkflowPage/ExecuteWorkflowPage.tsx

```diff
@@ -27,154 +30,187 @@ import {
   executeWorkflowWithBusinessKeyRouteRef,
   workflowInstanceRouteRef,
 } from '../../routes';
-import { BaseOrchestratorPage } from '../BaseOrchestratorPage/BaseOrchestratorPage';
-import { OrchestratorSupportButton } from '../OrchestratorSupportButton/OrchestratorSupportButton';
-import { WorkflowDialog } from '../WorkflowDialog';
-import { EditorViewKind } from '../WorkflowEditor';
-import { TitleFieldTemplate } from './TitleFieldTemplate';
-
-const WrappedForm = withTheme(require('@rjsf/material-ui-v5').Theme);
+import { getErrorObject } from '../../utils/error';
+import { BaseOrchestratorPage } from '../next/BaseOrchestratorPage';
+import SubmitBtn from '../SubmitBtn/SubmitBtn';
+import StepperForm from './StepperForm';
 
-interface ExecuteWorkflowPageProps {
+export interface ExecuteWorkflowPageProps {
   initialState?: Record<string, JsonValue>;
 }
 
-export const ExecuteWorkflowPage = (props: ExecuteWorkflowPageProps) => {
+const JsonTextAreaForm = ({
+  isExecuting,
+  handleExecute,
+}: {
+  isExecuting: boolean;
+  handleExecute: (
+    getParameters: () => Record<string, JsonValue>,
+  ) => Promise<void>;
+}) => {
+  const [jsonText, setJsonText] = React.useState('');
```

Comment:
Maybe the initial state should be `{}`?

Reply:
What if the JSON editor starts with this simple skeleton?

<img width="354" alt="image" src="https://github.com/caponetto/backstage-plugins/assets/638737/ccbb58d5-a67c-4a58-8dc5-1be48ac7c1c3">